ui/packages/consul-ui/translations/components/consul/en-us.yaml

```diff
@@ -117,8 +117,12 @@ intention:
 transparent-proxy: Transparent Proxy
 topology-metrics:
   source-type:
-    tooltip: This connection was defined in a proxy registration.
-    text: Defined in proxy registration
+    proxy-registration:
+      tooltip: This connection was defined in a proxy registration.
+      text: Defined in proxy registration
+    routing-config:
+      tooltip: This is not a registered Consul service. It’s a routing configuration that routes traffic to real services in Consul.
+      text: Routing Configuration
```

Comment:
Should these be:

```
proxy-registration:
      text: Defined in proxy registration
routing-config:
      text: Defined in Routing Configuration
```

or alternatively:

```
proxy-registration:
      text: Proxy Registration
routing-config:
      text: Routing Configuration
```

I think my question is how come on has `Defined by` and one doesn't? Not a block but just wondered.

Reply:
oh, let me ask design.

Reply:
Design would for `Routing configuration` to stay. The text would also be too long with `Defined in`.